versions.json

```diff
@@ -1 +1 @@
-["v3.29.4", "v4.10.2"]
+[]
```

Comment:
@fanciulli @Eomm i have opened a [pr](https://github.com/fastify/website-next/pull/37) #37  to address this based on the fastify current versions...hopefully that would solve the issue you mentioned in the in the pr description about an error that was popping up.

Reply:
@HassanBahati very nice, thank you. I will merge it as soon as PR is accepted

Reply:
Not doing anything since it seems to me that PR is changing the file to same exact value